lib/utils/environment.ts

```diff
@@ -74,7 +75,8 @@ export function BuildCommand(command: string, args?: string[], options?: Command
  * @throws        An error containing a message from the failed command's `stderr` output.
  */
 export function ExecuteCommand(command: Command): Promise<{}> {
-	let tries = ((command.options || {}).retries || 0) + 1;
+	let tries = (command.options.retries || 0) + 1;
+	let delay = command.options.delay || 1000;
```

Comment:
I could. But it'll be doing exactly the same thing with more work. :) This is where I feel it's style over substance, really (then again, if this was `((thing || {}).other || {}).thing || {}`, I'd use it as it were shorter.